src/Microsoft.Identity.Abstractions/DownstreamRestApi/DownstreamRestApiOptions.cs

```diff
@@ -3,106 +3,37 @@
 
 using System;
 using System.Collections.Generic;
-using System.ComponentModel;
 using System.Net.Http;
 
 namespace Microsoft.Identity.Abstractions
 {
     /// <summary>
     /// Options passed-in to call downstream web APIs.
     /// </summary>
-    public class DownstreamRestApiOptions
+    public class DownstreamRestApiOptions : AuthorizationHeaderProviderOptions
     {
-        AcquireTokenOptions _acquireTokenOptions = new();
-        HttpMethod _httpMethod = HttpMethod.Get;
-        string _protocolScheme = "Bearer";
-
         /// <summary>
-        /// constructor.
+        /// Default constructor
         /// </summary>
         public DownstreamRestApiOptions()
         {
         }
 
         /// <summary>
-        /// Copy constructor for <see cref="DownstreamRestApiOptions"/>
+        /// Copy constructor
         /// </summary>
-        protected DownstreamRestApiOptions(DownstreamRestApiOptions other)
+        /// <param name="other"></param>
+        public DownstreamRestApiOptions(DownstreamRestApiOptions other) : base(other)
         {
-            if (other == null)
-                throw new ArgumentNullException(nameof(other));
-
-            BaseUrl = other.BaseUrl;
-            CustomizeHttpRequestMessage = other.CustomizeHttpRequestMessage;
-            HttpMethod = other.HttpMethod;
-            ProtocolScheme = other.ProtocolScheme;
-            RelativePath = other.RelativePath;
-            AcquireTokenOptions = other.AcquireTokenOptions.Clone();
             Scopes = other.Scopes;
+            Serializer = other.Serializer;
+            Deserializer = other.Deserializer;
         }
 
-        /// <summary>
-        /// Base URL for the called downstream web API. For instance <c>"https://graph.microsoft.com/beta/"</c>.
-        /// </summary>
-        public string? BaseUrl { get; set; }
-
-        /// <summary>
-        /// Path relative to the <see cref="BaseUrl"/> (for instance "me").
-        /// </summary>
-        public string RelativePath { get; set; } = string.Empty;
-
-        /// <summary>
-        /// HTTP method used to call this downstream web API (by default Get).
-        /// </summary>
-        [DefaultValue("Get")]
-        public HttpMethod HttpMethod
+        /// <inheritdoc/>
+        public override AuthorizationHeaderProviderOptions Clone()
```

Comment:
This should return a DownstreamRestApiOptions

Reply:
then that cannot be an `override` method, this would be to be a new method?

Reply:
Addressed